src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -1831,6 +1832,12 @@ public Microsoft.PowerShell.ExecutionPolicy ExecutionPolicy
         /// </summary>
         public bool ThrowOnRunspaceOpenError { get; set; } = false;
 
+        /// <summary>
+        /// If not null, the working directory of the runspace is set to this path. If null,
+        /// the <see cref="Environment.CurrentDirectory">process working directory</see> is used as a default.
+        /// </summary>
+        public string WorkingDirectory { get; set; } = null;
```

Comment:
As this is a PowerShell location this probably should be called `Location` or `DefaultLocation` (or something along those lines). The location can be more than just a filesystem path so the API should probably not tie itself to that naming standard.

Reply:
Good point. I used the name I mentioned in the original issue, but `DefaultLocation` is better.